ui/src/containers/SolutionList.js

```diff
@@ -180,7 +180,7 @@ const SolutionsList = (props) => {
             {isEnvironmentPreparing && (
               <LoaderContainer>
                 <Loader size="small"></Loader>
-                {intl.translate('preparing_environemnt', {
+                {intl.translate('preparing_environment', {
```

Comment:
Should be in the same commit as the one changing the key in `src/translations/`.